swift/.swift-format.json

```diff
@@ -0,0 +1,69 @@
+{
+    "fileScopedDeclarationPrivacy": {
```

Comment:
Is this a standard swift style?

Reply:
This is all the default properties except `AlwaysUseLowerCamelCase` which I had to set to false due to our use of snake case and screaming case in the code.